code/datums/components/storage/concrete/tilebound.dm

```diff
@@ -0,0 +1,10 @@
+/datum/component/storage/concrete/tilebound
+	var/base_tile
+
+/datum/component/storage/concrete/tilebound/real_location()
+	return base_tile
+
+/datum/component/storage/concrete/tilebound/do_quick_empty(atom/_target)
+	if(!_target)
+		_target = real_location()
+	return ..(_target)
```

Comment:
```suggestion
	return ..()
```

You dont need to pass it along manually like that

Reply:
Wasn't sure if it was necessary or not given the potential change to the parameter. Good to know!